### ISSUE TITLE
Test M2 and M1 deployment with MICO

### ISSUE THREAD

**Issue by wagnerdk** [UST-MICO/mico#742]
Test M2 and M1 deployment with MICO and document all errors which appear as issues.

Depends on: #715, #732, #729

**Comment by davidkopp** [comment]
Note: Before the deployment of the message generator one environment variable have to be set:
```
KAFKA_BROKER_HOST: bootstrap.kafka
```

**Comment by davidkopp** [comment]
M1: https://github.com/UST-MICO/test_message_generator
M2: https://github.com/UST-MICO/msg-validator

**Comment by davidkopp** [comment]
Currently following setup is tested and works:
- [test_message_generator](https://github.com/UST-MICO/test_message_generator) deployed via MICO (required environment variables are set)
- [kafka-faas-connector](https://github.com/UST-MICO/kafka-faas-connector/) manually deployed:
  ```bash
  kubectl apply -f kafka-faas-connector.yml
  ```
- Consumer
  ```bash
  kubectl -n test-kafka exec -it $KAFKACAT -- kafkacat -b bootstrap.kafka:9092 -G mico -C -t transform-result
  ```

What's missing:
- Fix msg-validator (see issue #752) and deploy it with MICO
- Deploy kafka-faas-connector with MICO (see #744)
